Packages/com.unity.ide.rider.tests/Tests/Editor/CSProjectTests.cs

```diff
@@ -216,24 +217,79 @@ public void AllowUnsafeBlock()
                 var csprojFileContents = m_Builder.ReadProjectFile(m_Builder.Assembly);
                 StringAssert.Contains("<AllowUnsafeBlocks>True</AllowUnsafeBlocks>", csprojFileContents);
             }
+
+            [TestCase(new Object[] {"C:/Analyzer.dll"})]
+            [TestCase(new Object[] {"C:/Analyzer.dll", "C:/Analyzer2.dll"})]
```

Comment:
Passing actual content of csc.rsp would be more flexible. It may be:
In one case: ```
-a:C:\Work\com.unity.ide.rider\Test\Data\ProjectGenerationTests\RoslynAnalyzers\analyzers\dotnet\cs\Comments_analyser.dll;C:\Work\com.unity.ide.rider\Test\Data\ProjectGenerationTests\RoslynAnalyzers\analyzers\dotnet\cs\UnityEngineAnalyzer.dll
-a:Comments_analyser1.dll
```
In second case just an empty string.
In 3rd case just:
`-a:Comments_analyser1.dll`

Reply:
Wouldn't we rather have separate tests for the parser?

Reply:
Ok, fair enough. Then just use `new string[]` instead of `new Object[]`.

Reply:
I tried to use `new string[]` however NUnit didn't like it, perhaps there's some other way. I'll try again.

Reply:
Oh, sorry. I see you can only do `object` instead of `Object` there.